tests/Shared/RepoTesting/Directory.Packages.Helix.props

```diff
@@ -30,24 +41,53 @@
+    <PackageVersion Include="Aspire.Hosting.Python" Version="$(PackageVersion)" />
+    <PackageVersion Include="Aspire.Hosting.Qdrant" Version="$(PackageVersion)" />
     <PackageVersion Include="Aspire.Hosting.RabbitMQ" Version="$(PackageVersion)" />
     <PackageVersion Include="Aspire.Hosting.Redis" Version="$(PackageVersion)" />
+    <PackageVersion Include="Aspire.Hosting.Sdk" Version="$(PackageVersion)" />
     <PackageVersion Include="Aspire.Hosting.Seq" Version="$(PackageVersion)" />
     <PackageVersion Include="Aspire.Hosting.SqlServer" Version="$(PackageVersion)" />
+    <PackageVersion Include="Aspire.Hosting.Testing" Version="$(PackageVersion)" />
     <PackageVersion Include="Aspire.Hosting.Valkey" Version="$(PackageVersion)" />
+
     <PackageVersion Include="Aspire.Microsoft.Azure.Cosmos" Version="$(PackageVersion)" />
     <PackageVersion Include="Aspire.Microsoft.Data.SqlClient" Version="$(PackageVersion)" />
     <PackageVersion Include="Aspire.Microsoft.EntityFrameworkCore.Cosmos" Version="$(PackageVersion)" />
     <PackageVersion Include="Aspire.Microsoft.EntityFrameworkCore.SqlServer" Version="$(PackageVersion)" />
     <PackageVersion Include="Aspire.Milvus.Client" Version="$(PackageVersion)" />
     <PackageVersion Include="Aspire.MongoDB.Driver" Version="$(PackageVersion)" />
     <PackageVersion Include="Aspire.MySqlConnector" Version="$(PackageVersion)" />
+    <PackageVersion Include="Aspire.NATS.Net" Version="$(PackageVersion)" />
     <PackageVersion Include="Aspire.Npgsql" Version="$(PackageVersion)" />
     <PackageVersion Include="Aspire.Npgsql.EntityFrameworkCore.PostgreSQL" Version="$(PackageVersion)" />
     <PackageVersion Include="Aspire.Oracle.EntityFrameworkCore" Version="$(PackageVersion)" />
     <PackageVersion Include="Aspire.Pomelo.EntityFrameworkCore.MySql" Version="$(PackageVersion)" />
+    <PackageVersion Include="Aspire.Qdrant.Client" Version="$(PackageVersion)" />
     <PackageVersion Include="Aspire.RabbitMQ.Client" Version="$(PackageVersion)" />
+    <PackageVersion Include="Aspire.Seq" Version="$(PackageVersion)" />
     <PackageVersion Include="Aspire.StackExchange.Redis" Version="$(PackageVersion)" />
+    <PackageVersion Include="Aspire.StackExchange.Redis.DistributedCaching" Version="$(PackageVersion)" />
+    <PackageVersion Include="Aspire.StackExchange.Redis.OutputCaching" Version="$(PackageVersion)" />
+
+    <PackageVersion Include="Microsoft.Extensions.ServiceDiscovery" Version="$(PackageVersion)" />
+    <PackageVersion Include="Microsoft.Extensions.ServiceDiscovery.Abstractions" Version="$(PackageVersion)" />
+    <PackageVersion Include="Microsoft.Extensions.ServiceDiscovery.Dns" Version="$(PackageVersion)" />
+    <PackageVersion Include="Microsoft.Extensions.ServiceDiscovery.Yarp" Version="$(PackageVersion)" />
+  </ItemGroup>
+
+  <ItemGroup Label="For tests">
+    <PackageVersion Include="xunit" Version="$(XunitVersion)" />
```

Comment:
Why are the xunit packages needed to be defined here, but not any other external packages?

Reply:
When a test project is built in the repo, Arcade adds these references automatically. But when we are building outside the repo, the Arcade targets are not used, thus we need to explicitly add these. 

Any other external packages needed by a project would be added by those projects themselves. We add `PackageVersion` for them in `./tests/Shared/RepoTesting/Directory.Packages.Helix.props`. Hm this should be added in a common `README.md` too.